### PR TITLE
Issue 5

### PR DESCRIPTION
**Pull request by drkfmorton**
Updated readme file
Added python framework to manage code and functionality

### CODE REVIEW

**Comment by jonmarcw** [comment]
I'm having to get up to speed to Django, I didn't even have the module installed. Let's add a requirements.txt file and put in the modules that are required to run the django server and xavier. Unless that gets automated within the django framework.

**Comment by drkfmorton** [comment]
@jatpdubya I thought it was auto-generated, and forgot to check before I committed. Let me check on this.

**Comment by jonmarcw** [comment]
@drkfmorton When I tried to run as per the README, I got a missing Django module error, so I ran pip install django and then followed the README.
Although, I did python3 ./manage.py runserver by default because my python path is still python 2, when running as-is I got a syntax error. After installing django, python3 ./manage.py runserver works fine.

**Comment by drkfmorton** [comment]
@jatpdubya I added the setup logic. Please review, thanks.

File: xavier/xavier/setup.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Installing dependencies..."
+python3 -m pip install -r requirements.txt
+echo "Done installing dependencies"
+
+echo "Set Python Path"
+
```
Comment (jonmarcw) [inline xavier/xavier/setup.sh:8]: Should we set up the virtualenv prior to intalling the requirements, so they all get installed within the specific python environment container that we create?

**Review by jonmarcw — APPROVED** [review]

**Review by Sclayton1981 — APPROVED** [review]

**Pull request merged**